project-repo/code-practices.md

```diff
@@ -0,0 +1,41 @@
+# Code practices
+
+- All new code comes with tests. Fixes not coming with tests is rather the exception than the norm.
+
+- We follow upstream/community formatting conventions for the given technology. For instance. in Go, we tighten it using `gofumt` instead of `gofmt`. In Rust, `rustfmt`.
```

Comment:
I think those 2 parts are actually separated. Especially the use of `gofumt` (which is stricter than `gofmt`, but compatible: what pass gofumt will always pass gofmt. However, not all "gofmted" code will necessarily pass gofumt).